doc/cephfs/client-auth.rst

```diff
@@ -316,8 +316,8 @@ Changing rw permissions in caps
 It's not possible to modify caps by running ``fs authorize`` except for the
 case when read/write permissions have to be changed. This so because the
 ``fs authorize`` becomes ambiguous. For example, user runs ``fs authorize
```

Comment:
This section is missing a number of definite and indefinite articles and will have to be written conditionally.

Reply:
@zdover23 I'm merging this change. @yehaifeng addressed your pervious comment and this comments probably needs a doc refactor.

Reply:
@vshankar: Go for it. I'll give it a grammar pass later today.